yourgov/components/FormMobileFoodVendorPermit/utils.ts

```diff
@@ -31,3 +33,29 @@ export const yupPhoneField = yup
 			"Mobile numbers must begin with '04', landline numbers must begin with an area code",
 	})
 	.length(10, 'Phone number must be 10 digits');
+
+export function hasMultipleErrors(errors: FieldValues) {
+	return Object.keys(errors).length > 1;
+}
+
+export const formHomePage =
+	'/app/licences-and-permits/apply/mobile-food-vendor-permit/form';
```

Comment:
`form` is the name of one of our pages?

Reply:
Yeah, it's the main Task list page https://design-system.agriculture.gov.au/pr-preview/pr-1734/yourgov/app/licences-and-permits/apply/mobile-food-vendor-permit/form

Reply:
Ok, no worries. Just seems an odd name for a page. I've never seen it before.

Reply:
Could probably work on better names for some of these routes later if we feel it has value